mqgo/src/meqa/mqgen/main.go

```diff
@@ -29,19 +29,34 @@ func main() {
 	swaggerFile := flag.String("s", swaggerJSONFile, "the swagger.yml file location")
 	algorithm := flag.String("a", "all", "the algorithm - simple, object, path, all")
 	verbose := flag.Bool("v", false, "turn on verbose mode")
+	whitelistFile := flag.String("w", "", "the whitelist.txt file location")
 
 	flag.Parse()
-	run(meqaPath, swaggerFile, algorithm, verbose)
+	run(meqaPath, swaggerFile, algorithm, verbose, whitelistFile)
 }
 
-func run(meqaPath *string, swaggerFile *string, algorithm *string, verbose *bool) {
+func run(meqaPath *string, swaggerFile *string, algorithm *string, verbose *bool, whitelistFile *string) {
 	mqutil.Verbose = *verbose
 
 	swaggerJsonPath := *swaggerFile
 	if fi, err := os.Stat(swaggerJsonPath); os.IsNotExist(err) || fi.Mode().IsDir() {
 		fmt.Printf("Can't load swagger file at the following location %s", swaggerJsonPath)
 		os.Exit(1)
 	}
+	whitelistPath := *whitelistFile
```

Comment:
Why do we need a different assignment?
We are directly using the value without any changes.